frontend/src/components/navbar/navbar.css

```diff
@@ -0,0 +1,132 @@
+
+.logo {
+  height: 50%;
+  size: 50%;
+  margin-left: 20px;
+}
+
+/* .navbar-button:hover {
```

Comment:
@bqh5026 The reason hover isn't working is that lines 8-10 in `navbar.css` are commented out. Remove the commenting and it will work.

Reply:
We don't need margin-left for logo?